src/main/java/org/jabref/gui/DeleteDialog.java

```diff
@@ -0,0 +1,129 @@
+package org.jabref.gui;
+
+import javax.swing.JButton;
+import javax.swing.JDialog;
+import javax.swing.JPanel;
+import javax.swing.KeyStroke;
+
+import org.jabref.logic.l10n.Localization;
+import org.jabref.model.groups.GroupTreeNode;
+import javafx.collections.ObservableList;
+
+import javax.swing.BoxLayout;
+import javax.swing.JLabel;
+import javax.swing.JComponent;
+
+import java.awt.event.ActionListener;
+import java.awt.event.KeyEvent;
+import java.util.Iterator;
+import java.util.Set;
+import java.awt.event.ActionEvent;
+
+public class DeleteDialog extends JDialog {
+
+	public enum DeleteResult {
+		DELETE_FROM_DATABASE,
+		REMOVE_FROM_GROUPS,
+		CANCEL
+	}
+
+	protected DeleteResult result = DeleteResult.CANCEL;
+
+	/**
+	 * Create the dialog.
+	 * @param selectedGroups 
+	 * @param allEnriesGroups 
+	 */
+	public DeleteDialog(ObservableList<GroupTreeNode> selectedGroups, Set<String> allEnriesGroups) {
+		setTitle("Delete");
+		setModal(true);
+		getContentPane().setLayout(new BoxLayout(getContentPane(), BoxLayout.Y_AXIS));
+
+		JPanel panel = new JPanel();
+		getContentPane().add(panel);
+
+		String message = "";
+		if (selectedGroups.size() > 0) {
+			message += Localization.lang("Currently_there_are_selected_group(s)")+": <br/><ul>";
+			for(int i = 0; i < selectedGroups.size(); i++){
+				message += "<li>"+selectedGroups.get(i).getName() + "</li>"; 
+			}
+			message += "</ul>" + Localization.lang("If_you_would_like_to_remove_selected_entities_from_selected_groups_<br/>press_button") 
```

Comment:
Suggestion `Do_you_want_to_remove_the_selected_entries_from_the_selected_groups`

Reply:
The suggested text doesn't give message to user about action that should or should not be done.